lite-rpc/src/main.rs

```diff
@@ -196,7 +208,20 @@ pub async fn start_lite_rpc(args: Config, rpc_client: Arc<RpcClient>) -> anyhow:
         let inmemory_account_storage: Arc<dyn AccountStorageInterface> =
             Arc::new(InmemoryAccountStore::new());
         const MAX_CONNECTIONS_IN_PARALLEL: usize = 10;
-        let account_service = AccountService::new(inmemory_account_storage);
+        let (account_notification_sender, _) = tokio::sync::broadcast::channel(256);
+
+        let account_storage = if enable_accounts_on_demand_accounts_service {
+            Arc::new(AccountsOnDemand::new(
```

Comment:
is that correct? the AccountsOnDemand is an "add-on" on top if the main storage (in memory here) and uses a delegation pattern to implement a smart adapter with geyser

Reply:
Yes